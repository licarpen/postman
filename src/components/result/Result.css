.Result{
  grid-area: result;
  margin: auto;
  margin-top: 10px;
  width: 90%;
  
  .title{
    font-family: Arial, Helvetica, sans-serif;
    text-align: center;
    font-size: 2vw;
  }
  .results{
    border: lightslategrey 2px solid;
    background: rgb(246, 245, 245);
    overflow: scroll;
    height: 40vw;
  }
}