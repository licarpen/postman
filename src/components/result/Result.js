import React from 'react';
import PropTypes from 'prop-types';
import styles from './Result.css';

const Result = ({ result }) => (
  <div className={styles.Result}>
    <pre>{result}</pre>
  </div>
);

Result.propTypes = {
  result: PropTypes.string.isRequired
};

export default Result;
