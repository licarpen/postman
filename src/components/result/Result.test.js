import React from 'react';
import { shallow } from 'enzyme';
import Result from './Result';

describe('Result component', () => {
  it('matches a snapshot', () => {
    const wrapper = shallow(<Result result="sample result"/>);
    expect(wrapper).toMatchSnapshot();
  });
});
