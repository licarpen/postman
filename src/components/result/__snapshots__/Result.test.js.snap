// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Result component matches a snapshot 1`] = `
<div
  className="Result"
>
  <pre>
    sample result
  </pre>
</div>
`;
